### ISSUE TITLE
Cheats dont work

### ISSUE THREAD

**Issue by FunkyPixie** [wootguy/Glitch-Life#24]
I try to enable the cheats in the config but when i try to enable the cheats they dont work

**Comment by atmogenic** [comment]
yep, just came back to sven coop after being away for a year and now i can't enable the cheats no matter what i do

**Comment by FunkyPixie** [comment]
my theory is that it’s due to a sven coop update that broke the cheats, but that’s just my theory

**Comment by wootguy** [comment]
^ I think you're right. In the meantime there's this plugin which can enable cheats for everyone with `.cheats 1`
https://github.com/wootguy/Cheats
no way to control which cheats are enabled though.

**Comment by atmogenic** [comment]
the cheats you linked there don't seem to be working for me, i've moved the plugins from both svencoop-addon and just svencoop and edited the default_plugin to add the 

"plugin"
	{
		"name" "Cheats"
		"script" "Cheats"
	}
but nothing has changed, I type in one of the cheat commands and it just says "unknown command"

![notepad_zx7W2JEBBs](https://user-images.githubusercontent.com/56037304/111779482-1997d500-88ae-11eb-82c2-ade11177281f.png)

![explorer_ylPtqganbM](https://user-images.githubusercontent.com/56037304/111779536-29afb480-88ae-11eb-923e-49cdc8edf6db.png)

**Comment by wootguy** [comment]
`//` tells the game to ignore whatever comes after that. Remove the `//`s before the Cheats plugin block to fix.

**Comment by atmogenic** [comment]
ah ok I'll try that out
